frontend/src/pages/Login.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import "../App.css"
+
+function Login() {
+  return (
+    <div className='Login-Page'>
```

Comment:
I would suggest to be consistent with the naming conventions for CSS class here, `Login-Page` `Login-container` `Login-form`. Here you should be consistent. 
either go with `login-page` or `Login-Page`.

Reply:
Thank you for your feedback, I have fixed this in the next commit.